pkg/types/ccip/models.go

```diff
@@ -18,3 +63,40 @@ type TxMeta struct {
 	TxHash                  string
 	LogIndex                uint64
 }
+
+// EIP55 converts the provided address to EIP55.
+//
+// If the ith digit is a letter (ie. it’s one of abcdef) print it in uppercase if the 4*ith bit of the hash of the
+// lowercase hexadecimal address is 1 otherwise print it in lowercase.
+func EIP55(addr string) (string, error) {
```

Comment:
The "standard" implementation of this in Go is go-ethereum's `checksumHex`: https://github.com/ethereum/go-ethereum/blob/e31709db6570e302557a9bccd681034ea0dcc246/common/types.go#L261

Reply:
I will also have to copy some more stuff if I copy this.
```
BytesToAddress(FromHex(s))
```

I think the difference is only in performance but current implementation is cleaner.